lib/WebpackConfig.js

```diff
@@ -222,6 +224,11 @@ class WebpackConfig {
         this.useReact = true;
     }
 
+    enableVueLoader(options) {
+        this.useVueLoader = true;
+        this.vueOptions = options;
+    }
```

Comment:
Can you change `options` to be a callback, like we do with `configureBabel()`? The reason is that we *will* (in order to extract styles) add a bunch of options to the loader automatically. If the user wants to add even *more* options, we'll want them to do that in callback (we'll pass them our options, then they can change/add them).